rotkehlchen/globaldb/upgrades/manager.py

```diff
@@ -8,7 +8,13 @@
 from rotkehlchen.utils.misc import ts_now
 from rotkehlchen.utils.upgrades import UpgradeRecord
 
-from ..utils import GLOBAL_DB_VERSION, MIN_SUPPORTED_GLOBAL_DB_VERSION, _get_setting_value
+from ..utils import (
+    GLOBAL_DB_VERSION,
+    MIN_SUPPORTED_GLOBAL_DB_VERSION,
+    _add_setting_value,
+    _delete_setting_value,
```

Comment:
importing `_` functions? Wtf? Do you even know what `_` means? It's supposed to be private to the module. This is silly.